src/fidesops/ops/schemas/connection_configuration/connection_config.py

```diff
@@ -32,6 +33,16 @@ class Config:
         orm_mode = True
         use_enum_values = True
         extra = Extra.forbid
+        schema_extra = {
+            "example": {
+                "name": "Foo",
+                "key": FidesOpsKey("eb65d06059de1c30aa2b973a33075"),
```

Comment:
Hey @wiseaidev, thanks for the contributions!

These values will end up in our autogenerated OpenAPI documentation, so we prefer they're kept human readable. e.g.  `connection_config_key` would be more appropriate here.